docs/learn-faq.md

```diff
@@ -81,6 +81,88 @@ This script helps you quickly identify which validator is the lowest-staked and
 
 It is also possible to copy both `own stake` and `other stake` on the Staking page to a spreadsheet and then do a simple calculation.
 
+## Myths
+
+Below we present some myths about Polkadot that we've seen in the community.
+We're not absolutely sure what the origin of these myths are -- maybe they arise
+from people who have not taken the time to understand Polkadot. Regardless,
+we have collected a rebuttal for each of the common myths below. Feel free to
+link out to this page if you see one of these myths propagated on discussion platforms
+of the internet. 
+
+### Polkadot is an "Ethereum-killer"
+
+Some people position blockchains antagonistically against each other, claiming
+that all of the new blockchains are by default "Ethereum killers". The line of
+thinking is that there is a limited market share for blockchain platforms and
+therefore all projects are competing and vying for the lion's share. There may
+be some truth to this in the wide ecosystem of blockchain projects, but not all
+of them.
+
+Polkadot and Ethereum have significant differences in their technology, governance
+ideology, and community that instead forms a complimentary whole instead of a
```

Comment:
```suggestion
ideology, and community that instead form a complimentary whole instead of a
```

since this could be rewritten as "the significant differences form..."